PocketSocket/PSWebSocketServer.m

```diff
@@ -48,6 +48,9 @@ @interface PSWebSocketServerConnection : NSObject
 @end
 @implementation PSWebSocketServerConnection
 
+// private member variable
+bool _isV6Address = false;
```

Comment:
This should have curly braces around it; otherwise it isn't an instance/member variable, it's a global variable.